lib/transaction/index.js

```diff
@@ -889,11 +889,14 @@ function acceptDistributedTraceHeaders(transportType, headers) {
     // assumes header keys already lowercase
     const tracestate = headers[TRACE_CONTEXT_STATE_HEADER]
     this.acceptTraceContextPayload(traceparent, tracestate, transport)
-  } else if (NEWRELIC_TRACE_HEADER in headers) {
+  } else if (headers[NEWRELIC_TRACE_HEADER]) {
     logger.trace('Accepting newrelic DT payload for transaction %s', this.id)
     // assumes header keys already lowercase
     const payload = headers[NEWRELIC_TRACE_HEADER]
     this._acceptDistributedTracePayload(payload, transport)
+  } else {
+    this.agent.recordSupportability('DistributedTrace/AcceptPayload/Ignored/Null')
```

Comment:
ah, i think now i see why the 'in' was used. this metric you are adding here is intended if the payload is null, just for the New Relic format. so if the 'newrelic' header exists, but there's nothing in the payload itself. 

so we aren't quite doing the right thing here if it is a string, this location handles both formats, and we prob should protect against it being null/undefined as well.

so now looking at this closer, we prob wanna just add some checking at the top. you could prob continue to use the `in` operator in that case or you could use `Object.hasOwnProperty`. hasOwn would be slightly more restrictive, as won't walk prototype chain but that should also be rare.

Reply:
Thanks! I'll add a check at the top instead and keep the `in` operator.